src/ontology/reports/sync-subClassOf.added.self_parentage_issues.tsv

```diff
@@ -0,0 +1,83 @@
+subject_mondo_id	subject_mondo_label	object_mondo_id	subject_source_id	object_source_id	object_mondo_label
```

Comment:
## Suggested action plan
- [ ] @matentzn @twhetzel @nicolevasilevsky Review and share thoughts

I think this [google sheet](https://docs.google.com/spreadsheets/d/1kjrzBGxGIirsMbHQWde3u8Z6YZRL0FyUSTM3eDWvEfI/edit#gid=324968868) needs to go through curation.

Copy/paste from [my comments](https://github.com/monarch-initiative/mondo-ingest/issues/400#issuecomment-2081684554) (I suggest taking a look as they have more detail) in https://github.com/monarch-initiative/mondo-ingest/issues/400:
> I think there multiple types of cases going on here:
> 1. In some cases, the suggestion from the "subclass sync - add" pipeline is correct, it might be that there are Mondo classes which need to get broken into 2 classes ;(
> 2. In other (hopefully most) cases, the suggestions are incorrect, and the pipeline needs to not make such suggestions.

> The reason that these suggestions are occurring is because they are instances where there is a legal `sub subClassOf obj` relationship in the source, but Mondo has done a proxy merge, so the same Mondo ID is mapped to both `sub` and `obj`.


> I think we need to go through the [google sheet](https://docs.google.com/spreadsheets/d/1kjrzBGxGIirsMbHQWde3u8Z6YZRL0FyUSTM3eDWvEfI/edit#gid=324968868) and parse out which of these cases that the pipeline is suggestion are correct and which are not. And I will probably need a file which lists all of Mondo's proxy merges so that I can have the pipeline read it and filter such suggestions out.

Reply:
[Nico opted not](https://github.com/monarch-initiative/mondo-ingest/pull/501#discussion_r1598101014) for this curation step.